README.md

```diff
@@ -9,17 +8,24 @@ An example output from OptView is shown below.
 
 ![Example](doc/OptView.png)
 
-Documentation
--------------
+## Optimizer Support
+pyOptSparse provides Python interfaces for a number of optimizers.
+However, we do not provide the source code for SNOPT and NLPQLP, due to their restrictive license requirements.
+Please contact the authors of the respective codes if you wish to obtain them.
```

Comment:
Tell me where to find the authors of the codes? Are they within the files?

Reply:
Links in the docs page for each optimizer, but will add a line redirecting readers there.